spec/models/form/sales/questions/uprn_confirmation_spec.rb

```diff
@@ -23,6 +23,10 @@
     expect(question.derived?(nil)).to be false
   end
 
+  it "has the correct unanswered_error_message" do
+    expect(question.unanswered_error_message).to eq("You must answer is this the right address?")
```

Comment:
Isn't this dependant on a translation now? If so, we shouldn't test like this because it stops people making changes just by changing the translations file.

Reply:
It is testing this in 2023 which I assume no one will change going forward, but I can update this to use the translation.

Reply:
Updated uprn_confirmation and uprn_known